toolkit/scripts/pkggen.mk

```diff
@@ -182,6 +183,8 @@ $(STATUS_FLAGS_DIR)/build-rpms.flag: $(cached_file) $(chroot_worker) $(go-schedu
 		$(if $(CONFIG_FILE),--base-dir="$(CONFIG_BASE_DIR)") \
 		$(if $(filter y,$(RUN_CHECK)),--run-check) \
 		$(if $(filter y,$(STOP_ON_PKG_FAIL)),--stop-on-failure) \
+		$(if $(filter n,$(USE_PACKAGE_BUILD_CACHE)),--no-cache) \
```

Comment:
Open question - Should a strict `n` be set to set these flags, or anything but a `y` ?

Reply:
my 2 cents - Anything but `y`. The option is to `USE_PACKAGE_BUILD_CACHE`, which seems like one should affirm its usage to get the benefit.

Reply:
The pattern I see most often is "y" is yes, everything else is no.
So here it would be `$(if $(filter n,$(USE_PACKAGE_BUILD_CACHE)), ,--no-cache)`

Reply:
Or change it to `AVOID_PACKAGE_BUILD_CACHE`